examples/tv-app/linux/include/application-basic/Application.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "gen/enums.h"
+#include <cstdint>
+#include <string>
+
+struct Application
+{
+    char vendorName[32]                  = "\0";
```

Comment:
"\0" is a double null terminator. Please use "" or {} 

https://en.cppreference.com/w/cpp/language/zero_initialization

Reply:
Updated.

Reply:
Can you please find-replace remaining instances of `"\0"` in the PR with `""`?

Reply:
Updated.